### ISSUE TITLE
Update Guide page pattern content

### ISSUE THREAD

**Issue by yatesn** [ONSdigital/design-system#1868]
Add guide page template for use in DS to document linear guidance

Add a layout to the DS that uses the guide page pattern so we can document more complex guidance in a simpler multi-page linear template.
![image](https://user-images.githubusercontent.com/97620554/151184258-b15be43f-258b-4dff-9abc-cd9329d85c1e.png)

**Comment by yatesn** [comment]
Await review from content designer

**Comment by jrbarnes9** [comment]
Already published, so this needs content design review.

**Comment by jrbarnes9** [comment]
https://trello.com/c/1wBGbk9Q/569-epic-design-system-patterns-where-there-is-little-or-no-documentation-or-it-needs-a-review

**Comment by OscarJackson** [comment]
Is this just text changes to the guidance page.

I've updated the guidance page text. I noticed that the link title doesn't match the destination on that, but I think the link title (import the files and assets) is better than the current page title (using the CSS, JS and assets).
Does anybody have any objections if I change this to the former? @JNATION85 @Majsol @alessioventuriniAND @precious-onyenaucheya-ons
